SPECS-EXTENDED/gupnp/gupnp.spec

```diff
@@ -14,11 +14,13 @@ Source0:        https://github.com/GNOME/%{name}/archive/refs/tags/%{name}-%{ver
 BuildRequires:  docbook-style-xsl
 BuildRequires:  gi-docgen
 %endif
+BuildRequires:  cmake
 BuildRequires:  gobject-introspection-devel
+BuildRequires:  gssdp-devel
 BuildRequires:  meson
 BuildRequires:  vala
 BuildRequires:  %{_bindir}/xsltproc
-BuildRequires:  pkgconfig(gssdp-1.6) >= %{gssdp_version}
```

Comment:
Do you know what changed that broke the previous version? Is this a workaround for a tooling shortcoming?

Reply:
After looking at the logs, it seem like the main issue was the unpackaged man1 file below:
```
time="2024-04-18T10:14:37Z" level=debug msg="RPM build errors:"
time="2024-04-18T10:14:37Z" level=debug msg="error: Installed (but unpackaged) file(s) found:"
time="2024-04-18T10:14:37Z" level=debug msg="   /usr/share/man/man1/gupnp-binding-tool-1.6.1.gz"
```

I think there may be no need to update the BRs.

Reply:
Yes that was the main fix - the man1 file is generated regardless of the docs condition. I will revert the BR changes